qemu/qemu.go

```diff
@@ -2490,6 +2493,15 @@ func (config *Config) appendGlobalParam() {
 	}
 }
 
+func (config *Config) appendPFlashParam() {
+	if len(config.PFlash) != 0 {
```

Comment:
this condition is not required, `for _, p := range config.PFlash ` should be enough

Reply:
thanks @devimc